packages/gestalt-codemods/next/box-replace-marginleft-marginright.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Converts
+ *  <Box marginLeft={...} marginRight={...} /> to <Box marginStart={...} marginEnd={...} />
+ */
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  let localIdentifierName;
+  let fileHasModifications = false;
+
+  src.find(j.ImportDeclaration).forEach((path) => {
+    const decl = path.node;
+    if (decl.source.value !== 'gestalt') {
+      return null;
+    }
+
+    localIdentifierName = decl.specifiers
+      .filter((node) => ['Box'].includes(node.imported.name))
+      .map((node) => node.local.name);
+    return null;
+  });
+
+  if (!localIdentifierName) {
+    return null;
+  }
+
+  const transform = src
+    .find(j.JSXElement)
+    .forEach((jsxElement) => {
+      const { node } = jsxElement;
+
+      if (!localIdentifierName.includes(node.openingElement.name.name)) {
+        return null;
+      }
+
+      const attrs = node.openingElement.attributes;
+
+      const newAppendAttr = [];
+      const newAttrs = attrs
+        .map((attr) => {
+          if (attr?.name?.name && attr.name.name === 'marginLeft') {
```

Comment:
can we include responsive variations?

Reply:
This would not remove 
```
  'smMarginLeft',
  'mdMarginLeft',
  'lgMarginLeft',
  'smMarginRight',
  'mdMarginRight',
  'lgMarginRight',
```

Reply:
It's a codemod, so I wouldn't go very sophisticated, copy-paste `if{...}` for each case?

Reply:
Psh, not very sophisticated! No way! 😅